echoregions/regions2d/regions2d.py

```diff
@@ -551,6 +552,9 @@ def transect_mask(
         bbox_distance_threshold: float
             The maximum value for how far apart the left and right bounding box for each transect
             value region. Default is set to 1 minute.
+        must_pass_check : bool
+            To determine whether fail if transect sequence check strings exist or pass and print
+            the transect sequence checks.
 
         Returns
         -------
```

Comment:
Right now the docstring for the return `M` is below:
```
        M : Data Array
            A DataArray masked by the transect values from the Regions2d.data dataframe
            with dimensions (ping_time, depth).
```
Could you re-write it so that it is clear what is 1 and what is 0 in the output?
Something like: "A binary DataArray containing ... "

Reply:
Made this change.